core/templates/pages/story-editor-page/editor-tab/story-editor.directive.html

```diff
@@ -88,17 +88,17 @@ <h3 class="story-card-header">Story Card</h3>
             </div>
             <div class="story-thumbnail">
               <label class="form-heading">Thumbnail Image*</label>
-              <thumbnail-uploader filename="story.getThumbnailFilename()"
-                                  update-filename="updateStoryThumbnailFilename"
-                                  bg-color="story.getThumbnailBgColor()"
-                                  update-bg-color="updateStoryThumbnailBgColor"
-                                  allowed-bg-colors="allowedBgColors"
-                                  aspect-ratio="'4:3'"
-                                  preview-title="editableTitle"
-                                  preview-description="editableDescription"
-                                  preview-description-bg-color=""
-                                  preview-footer="">
-              </thumbnail-uploader>
+              <oppia-thumbnail-uploader [filename]="story.getThumbnailFilename()"
+                                        [update-filename]="updateStoryThumbnailFilename"
+                                        [bg-color]="story.getThumbnailBgColor()"
+                                        [update-bg-color]="updateStoryThumbnailBgColor"
+                                        [allowed-bg-colors]="allowedBgColors"
+                                        [aspect-ratio]="'4:3'"
```

Comment:
If this property have static value for all occurances of `oppia-thubnail-uploader` tags. Then assign this variable in the component itself. Why create a input?